src/drivers/imu/invensense/icm42688p/ICM42688P.cpp

```diff
@@ -462,38 +506,40 @@ uint16_t ICM42688P::FIFOReadCount()
 	return combine(fifo_count_buf[1], fifo_count_buf[2]);
```

Comment:
can change the counter endian by setting INTF_CONFIG0:FIFO_COUNT_ENDIANbit. so that can be simplified by setting that bit during initialize.
that feature **has not** tested on ICM-42605

Reply:
Thanks for raising that idea, I'll give it a quick try on the icm42688p.

Reply:
Tested in https://github.com/PX4/Firmware/pull/15511.